### PR TITLE
migrated your code to python 3 with minor changes in libraries

### PR DESCRIPTION
**Pull request by maheshj01**

### CODE REVIEW

File: download.py [hunk 1]
```diff
@@ -7,51 +7,51 @@
...
         movieNm = row[1]
         url = row[2]
-        print "Working on ",movieNm,"..."
+        print("Working on ",movieNm,"...")
         cur1 = conn.cursor()
         cur1.execute('SELECT song FROM Songs WHERE movie_id = ?',(movie,))
         if not cur1.fetchone():
-            content = myopener.open(url).read()
+            content = requests.get(url)
             links = re.findall('<td class="songs-bitrate-1"><i class="fa fa-music"></i>(.+?)<em>',content)
```
Comment (amanvishnani) [inline download.py]: change `content` to `content.text` in `re.findall()` second parameter.

File: download.py [hunk 2]
```diff
@@ -65,47 +65,46 @@ def getAllMoviesList():
...
+    # class MyOpener(urllib.FancyURLopener):
+    #     version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
+    # myopener = MyOpener()
     for page in pages:
-        print "Started List for ", page
+        print("Started List for ", page)
         url = 'https://www.songspk.cloud/indian_movie/'+page+'_List.html'
-        content = myopener.open(url)
+        content = requests.get(url)
         tags = re.findall('<li class="list">.*?</li>',content.read())
```
Comment (amanvishnani) [inline download.py]: change `content.read()` to `content.text`

File: download.py [hunk 3]
```diff
@@ -7,51 +7,51 @@
...
     for row in cur:
         movie = row[0]
         url = row[1]
         if row[2] == None:
-            print "Working on ", movie
-            content = myopener.open(url).read()
+            print("Working on ", movie)
+            content = requests.get(url)
             try:
                 cover = re.findall('<img data-retina="" src="(.+?)"',content)[0]
```
Comment (amanvishnani) [inline download.py:21]: change `content` to `content.text`

File: download.py [hunk 4]
```diff
@@ -65,47 +65,46 @@ def getAllMoviesList():
...
-    class MyOpener(urllib.FancyURLopener):
-        version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
-    myopener = MyOpener()
+    # class MyOpener(urllib.FancyURLopener):
+    #     version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
+    # myopener = MyOpener()
     for page in pages:
-        print "Started List for ", page
+        print("Started List for ", page)
         url = 'https://www.songspk.cloud/indian_movie/'+page+'_List.html'
```
Comment (amanvishnani) [inline download.py]: change `www.songspk.cloud` to `www.songspk.run`

**Review by amanvishnani — CHANGES_REQUESTED** [review]
Please find below changes and correct them before I merge this request.

1. Add `requirements.txt` file with `requests` as a dependency in it.
2. Please find changes in the comment section of code

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 This Python Script will scrap out all the Songs URL from SongsPK and store it in a SQLite3 Database.
 ## Requirements
-- Python 2.x
+- Python 3.x
 
 ## How to run?
 - Clone this repository `git clone`
```
Comment (amanvishnani) [inline README.md:8]: After Clone this repository, add ` pip install -r requirements.txt`

**Review by amanvishnani — CHANGES_REQUESTED** [review]

**Comment by maheshj01** [comment]
I wii make the changes soon

**Comment by maheshj01** [comment]
> Please find below changes and correct them before I merge this request.
> 
> 1. Add `requirements.txt` file with `requests` as a dependency in it.
> 2. Please find changes in the comment section of code

I couldnt understand this I made rest of the changes but it gives an error content not found after changing content to content.text

File: download.py (hunk 1 above)
Comment (amanvishnani) [reply download.py]: Please Find the following changes..
```links = re.findall('<td class="songs-bitrate-1"><i class="fa fa-music"></i>(.+?)<em>',content.text)```
change content to content.text in re.findall() second parameter.

File: download.py (hunk 2 above)
Comment (amanvishnani) [reply download.py]: ``` tags = re.findall('<li class="list">.*?</li>',content.read())```
to 
``` tags = re.findall('<li class="list">.*?</li>',content.text)```

**Comment by amanvishnani** [comment]
Please read about [Requirements Files here](https://pip.pypa.io/en/stable/user_guide/#requirements-files)

> Logically, a Requirements file is just a list of pip install arguments placed in a file. 

I need this because you are using ```requests``` package which a user might not have.

Please Make changes to your master and its commit will be reflected here. You don't have to open a new pull request for that.

**Comment by maheshj01** [comment]
You can check the code now
two take aways for me from this project 

- `pip3 freeze > requirements.txt`
is much simpler to automatically add all the requirements for the project
- you don't need to submit a new pull request every time a commit to the forked repo synchronizes with the pull request xD

**Review by amanvishnani — APPROVED** [review]

**Comment by amanvishnani** [comment]
> You can check the code now
> two take aways for me from this project
> 
> * `pip3 freeze > requirements.txt`
>   is much simpler to automatically add all the requirements for the project
> * you don't need to submit a new pull request every time a commit to the forked repo synchronizes with the pull request xD

Yeah, ```pip3 freeze``` is a convenient command but we need to remove unwanted dependencies from it.
Anyways I will do it after merge.

**Comment by amanvishnani** [comment]
closes #4

**Pull request merged**